src/controller/trainee/routes.ts

```diff
@@ -8,35 +8,207 @@ import validation from './validation';
+ *        description: Enter Email and Password
+ *        required: true
+ *        content:
+ *           application/json:
+ *            schema:
+ *             type: object
+ *             required:
+ *              -email
+ *              -password
+ *             properties:
+ *               email:
+ *                type: string
+ *                example: 'john@successive.tech'
+ *               password:
+ *                type: string
+ *                example: 'john@123'
+ *     responses:
+ *         201:
+ *           description: Login Successful
+ */
+
+// post swagger
+/**
+ * @swagger
+ * /trainee:
+ *   post:
+ *     description: Create New trainee
+ *     tags: [Trainee]
+ *     requestBody:
+ *        description: Enter name,email,password,role to create new trainee
+ *        required: true
+ *        content:
+ *           application/json:
+ *            schema:
+ *             type: object
+ *             required:
+ *              -email
+ *              -password
+ *              -role
+ *              -name
+ *             properties:
+ *               name:
+ *                type: string
+ *                example: 'John Milton'
+ *               role:
+ *                type: string
+ *                example: 'trainee'
+ *               email:
+ *                type: string
+ *                example: 'john@successive.tech'
+ *               password:
+ *                type: string
+ *                example: 'john@123'
+ *     responses:
+ *         200:
+ *           description: Created new trainee successfully
+ */
+
+// Update Trainee
+/**
+ * @swagger
+ * /trainee/:id:
+ *   put:
+ *     description: Update existing trainee
+ *     tags: [Trainee]
+ *     consumes:
+ *         - application/json
+ *     produces:
+ *         - application/json
+ *     parameters:
+ *         - in: path
+ *           name: id
+ *           schema:
+ *            type: string
+ *           required: true
+ *           description: originalId of the trainee
+ *           example: 1234frg43455
+ *     requestBody:
+ *        description: Enter name,email,password,role to update trainee
+ *        required: true
+ *        content:
+ *           application/json:
+ *            schema:
+ *             type: object
+ *             properties:
+ *               name:
+ *                type: string
+ *                example: 'John hardy'
+ *               role:
+ *                type: string
+ *                example: 'trainee'
+ *               email:
+ *                type: string
+ *                example: 'hardy@successive.tech'
+ *               password:
+ *                type: string
+ *                example: 'hardy@123'
+ *     responses:
+ *         200:
+ *           description: trainee updated successfully
+ */
+
+// delete swagger -
+/**
+ * @swagger
+ * /trainee/:id:
+ *   delete:
+ *     description: Delete trainee
+ *     tags: [Trainee]
+ *     parameters:
+ *         - in: path
+ *           name: id
```

Comment:
wright swagger description and then route. DO not wright them separately.

Reply:
fixed